src/Hl7.Fhir.Base/Serialization/engine/ElementModelSerializationEngine.cs

```diff
@@ -83,7 +84,7 @@ private Resource deserialize(Func<ISourceNode> deserializer)
         }
 
 
-        public string SerializeToXml(Resource instance) => new CommonFhirXmlSerializer(_inspector).SerializeToString(instance);
+        public string SerializeToXml(Base instance) => new CommonFhirXmlSerializer(_inspector).SerializeToString(instance);
```

Comment:
```suggestion
[TemporarilyChanged]
        public string SerializeToXml(Base instance) => new CommonFhirXmlSerializer(_inspector).SerializeToString(instance);
```

Reply:
We should keep allowing this serialization, correct? There are use cases for this, so if we plan to factor out the old parsers and the engine only allows serialization of Resources, we have a gap!

Reply:
Yeah, it should stay on the XML serialization engine I guess.